cluster/saltbase/salt/fluentd-es/fluentd-es.yaml

```diff
@@ -12,6 +12,10 @@ spec:
     resources:
       limits:
         cpu: 100m
+        memory: 200Mi
+      requests:
+        cpu: 100m
+        memory: 200Mi
```

Comment:
Should we change the memory limit based on the machine size like we do for heapster?